samza-core/src/main/java/org/apache/samza/config/ClusterManagerConfig.java

```diff
@@ -85,20 +85,28 @@
   public static final String CLUSTER_MANAGER_CONTAINER_RETRY_COUNT = "cluster-manager.container.retry.count";
   private static final int DEFAULT_CONTAINER_RETRY_COUNT = 8;
 
-  /**
-   * Determines whether a JMX server should be started on the job coordinator
-   * Default: true
-   */
-  public static final String AM_JMX_ENABLED = "yarn.am.jmx.enabled";
-  public static final String CLUSTER_MANAGER_JMX_ENABLED = "cluster-manager.jobcoordinator.jmx.enabled";
-
   /**
    * The cluster managed job coordinator sleeps for a configurable time before checking again for termination.
    * The sleep interval of the cluster managed job coordinator.
    */
   public static final String CLUSTER_MANAGER_SLEEP_MS = "cluster-manager.jobcoordinator.sleep.interval.ms";
   private static final int DEFAULT_CLUSTER_MANAGER_SLEEP_MS = 1000;
 
+  /**
+   * Determines whether a JMX server should be started on job coordinator and container
+   * Default: true
+   */
+  public static final String JOB_JMX_ENABLED = "job.jmx.enabled";
+
+  /**
+   * Determines whether a JMX server should be started on the job coordinator
+   * Default: true
+   *
+   * @deprecated use {@code JOB_JMX_ENABLED} instead
+   */
+  public static final String AM_JMX_ENABLED = "yarn.am.jmx.enabled";
```

Comment:
Both of these static constants should be made private. Please correct me if i'm wrong, I don't see them being used anywhere outside of this class.
```suggestion
  private static final String AM_JMX_ENABLED = "yarn.am.jmx.enabled";
```